Mathlib/Tactic/Hint.lean

```diff
@@ -0,0 +1,132 @@
+/-
+Copyright (c) 2023 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import Std.Tactic.TryThis
+import Std.Linter.UnreachableTactic
+import Mathlib.Data.Nondet.Basic
+import Mathlib.Tactic.FailIfNoProgress
+import Mathlib.Mathport.Rename
+
+#align_import tactic.hint from "leanprover-community/mathlib"@"8f6fd1b69096c6a587f745d354306c0d46396915"
```

Comment:
I don't think we align tactic modules, right?

Reply:
Yes, I agree. @Komyyy added this one, but I don't think will object to removing it.